test/reporter.test.js

```diff
@@ -1,33 +1,45 @@
-/* global describe, it */
+/* global describe, before, it */
 'use strict'
 
 const bindings = require('../')
 const expect = require('chai').expect
 
 const env = process.env
+const maxIsReadyToSampleWait = 60000
```

Comment:
this is a really large number, does it make sense to wait for a whole minute or is it preferable to fail faster?

Reply:
It was set that way in some tests and now that `before` is used rather than a test case, it proliferated. We have a specific memory test where timeout had to be increased up to 120000 to prevent "flakiness". Not sure how this one plays out. Will dig deeper in next test iteration.

For posterity - this is where `liboboe` handles it: https://github.com/librato/oboe/blob/master/liboboe/settings.c#L1180